src/h5web/dataset-visualizer/VisToolbar.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Vis } from './models';
+import HeatmapToolbar from '../visualizations/heatmap/HeatmapToolbar';
+
+interface Props {
+  vis?: Vis;
+}
+
+function VisToolbar(props: Props): JSX.Element {
+  const { vis } = props;
+
+  if (vis === Vis.Heatmap) {
+    return <HeatmapToolbar />;
+  }
+  return <></>;
+}
+
+export default VisToolbar;
```

Comment:
This is the new component that will display the appropriate toolbar. All interactions will take place inside the toolbar of the respective visualisation.